src/com/tony/server/ResponseDeterminer.java

```diff
@@ -28,6 +29,8 @@ public Response determineResponse(Request request) {
             }
         } else if(uriList.contains(request.getURI())){
             return new MethodNotAllowedResponse();
+        } else if(request.getURI().startsWith("/parameters?")){
```

Comment:
what happens when a request comes in at  `/parameter` ? do you issue a 404? I feel like you shouldn't be checking to see if the `?` exists here.